### PR TITLE
Removing the notebooks from the github repository.

### PR DESCRIPTION
**Pull request by vincentpierre**
### Proposed change(s)

Deleting the notebooks from the repository.

### Useful links (Github issues, JIRA tickets, ML-Agents forum threads etc.)

[Jira-418](https://jira.unity3d.com/browse/MLA-418)

### Types of change(s)

- ~[ ] Bug fix~
- ~[ ] New feature~
- ~[ ] Code refactor~
- ~[ ] Breaking change~
- ~[ ] Documentation update~
- [x] Other (feature removal)

### Checklist
- ~[ ] Added tests that prove my fix is effective or that my feature works~
- [x] Updated the [changelog](https://github.com/Unity-Technologies/ml-agents/blob/master/com.unity.ml-agents/CHANGELOG.md) (if applicable)
- [x] Updated the [documentation](https://github.com/Unity-Technologies/ml-agents/tree/master/docs) (if applicable)
- [x] Updated the [migration guide](https://github.com/Unity-Technologies/ml-agents/blob/master/docs/Migrating.md) (if applicable)

### CODE REVIEW

File: ml-agents/setup.py [hunk 1]
```diff
@@ -56,7 +56,6 @@ def run(self):
         # Test-only dependencies should go in test_requirements.txt, not here.
         "grpcio>=1.11.0",
         "h5py>=2.9.0",
-        "jupyter",
         "matplotlib",
```
Comment (chriselion) [inline ml-agents/setup.py]: remove matplotlib too?

**Comment by chriselion** [comment]
I don't love notebooks in general. But I worry that this doesn't give us any way (that's under source control) to test the LL API or gym interface.

**Comment by vincentpierre** [comment]
By test, you mean testing or letting our users try it out? If it is testing, I think we should have a better way to do it. For letting our users try it out, I think we should have something but I don't know what yet.

**Comment by awjuliani** [comment]
My preference would be that we have something to replace these with. At the very least, ensure that the code is present within the docs.

**Comment by vincentpierre** [comment]
Isn't that what https://github.com/Unity-Technologies/ml-agents/blob/master/docs/Python-API.md and https://github.com/Unity-Technologies/ml-agents/blob/master/gym-unity/README.md are for. If we just move this code around, we will be facing the same problem : we cannot test this code.

**Comment by awjuliani** [comment]
@vincentpierre Okay. But we should make a Jira ticket for setting up a Colab notebook.

**Comment by vincentpierre** [comment]
MLA-809 if you want to add to it.

**Pull request merged**